donjo-app/controllers/Menu.php

```diff
@@ -158,6 +161,7 @@ public function ajax_add_sub_menu($tip = 1, $menu = '', $id = '')
 		$data['statistik_keluarga'] = $this->referensi_model->list_ref(STAT_KELUARGA);
 		$data['statistik_kategori_bantuan'] = $this->referensi_model->list_ref(STAT_BANTUAN);
 		$data['statistik_program_bantuan'] = $this->program_bantuan_model->list_program(0);
+		$data['kelompok'] = $this->kelompok_model->get_data_kelompok();
```

Comment:
@afa28 method `$this->kelompok_model->get_data_kelompok()` tidak ditemukan.

Reply:
Salah panggil, harusnya `$data['kelompok'] = $this->kelompok_model->list_data();`